appveyor.yml

```diff
@@ -1,17 +1,15 @@
-os: Visual Studio 2022
+os: Ubuntu2204
 
-before_build:
-  - nuget restore Renci.SshNet.sln
+services:
+- docker
 
-install:
-  - cinst dotnet-sdk --version=7.0.403 --limit-output
-
-build:
-  project: Renci.SshNet.sln
-  verbosity: minimal
+#for testing purposes only
+build_script:
+  - dotnet build test/Renci.SshNet.IntegrationTests/ -c Debug
 
 test_script:
-- cmd: >-
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net462\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
-    
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net7.0\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
+- mkdir artifacts
+- sudo tcpdump --list-interfaces
```

Comment:
```suggestion
- sudo tcpdump -i any -c1000 -w artifacts/tcpdump.pcap
```

Reply:
the artifacts directory is not connected yet.

Reply:
Oh

Reply:
I've added this

Reply:
It will wait for 1000 packets. Without the integration tests running, that might take a while 🙂